portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -33,9 +33,51 @@ definition {
 	@ignore = "true"
```

Comment:
Hi @AfonsoHSantos, please remove the ignore when the test is done :)

Reply:
fixed :)